File: src/Java/BehavioralPatterns/Mediator/CommunicationHub/src/Mediators/Group.java
package Mediators;

import Participants.Participant;

import java.util.ArrayList;
import java.util.List;

public class Group {
    private List<Participant> participants = new ArrayList<>();

    public boolean doesParticipantExist(Participant participant) {
        return participants.contains(participant);
    }

    public List<Participant> getParticipants() {
        return participants;
    }

    public void setParticipants(List<Participant> participants) {
        this.participants = participants;
    }
}

File: src/C#/BehavioralPatterns/Mediator/CommunicationHub/Mediators/Group.cs
using CommunicationHub.Participants;
using System;
using System.Collections.Generic;
using System.Text;

namespace CommunicationHub.Mediators
{
    public class Group
    {
        public List<Participant> Participants { get; set; } = new List<Participant>();

        public bool DoesParticipantExist(Participant participant) => Participants.Contains(participant);
    }
}
